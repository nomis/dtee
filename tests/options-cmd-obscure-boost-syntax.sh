#!/bin/bash
set -vx
. "$(dirname "$0")"/util/common.sh

# Check that obscure Boost syntax "--=..." is rejected ðŸ˜©
run_test_once "--debug-options" "-o" "stdout_append" "--=command" "-e" "stderr_append" "--=argument 1" "-c" "combined_append" "--=argument 2" "-q" "--=argument 3"
RET=$?

echo RET $RET
if [ $RET -eq 64 ]; then
	exit 0
fi
exit 1
