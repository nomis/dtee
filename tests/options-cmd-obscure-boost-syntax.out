dtee: unrecognised option '--=command'
Try 'dtee -h' for more information.
